rules/framework/vfs_overlay.bzl

```diff
@@ -62,20 +62,25 @@ def _build_subtrees(paths, vfs_prefix):
         # Assume the last bit is a file then add it as a file
         idx = 0
         for part in parts:
-            if idx == len(parts) - 1:
+            # Only use this value of curr_subdirs if it contains a dict
```

Comment:
I'm still trying to wrap my head around how this would exist but not be a dict - mainly that the upstream logic should be setting it to the right type. Is it safe to change the code to check for `and current_subdirs` ? e.g testing for `not None`?